### PR TITLE
Fix code reference sidebar

### PR DESCRIPTION
**Pull request by m-vdb**
**Proposed changes**:
- Fixes https://github.com/RasaHQ/engineering-backing/issues/12

**Status (please check what you already did)**:
- [ ] added some tests for the functionality
- [ ] updated the documentation
- [ ] updated the changelog (please check [changelog](https://github.com/RasaHQ/rasa/tree/master/changelog) for instructions)
- [ ] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa#code-style) for instructions)

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -150,7 +150,7 @@ coveralls = "^2.0.0"
 towncrier = "^19.2.0"
 toml = "^0.10.0"
 pep440-version-utils = "^0.3.0"
-pydoc-markdown = "^3.5.0"
+pydoc-markdown = "^3.5.1"
```
Comment (m-vdb) [inline pyproject.toml:153]: this requires https://github.com/NiklasRosenstein/pydoc-markdown/pull/156 to be merged and released to pypi

Comment (tmbo) [reply pyproject.toml:153]: that one seems to be released, should we update?

File: pyproject.toml [hunk 2]
```diff
@@ -145,7 +145,7 @@ coveralls = "^3.0.1"
 towncrier = "^19.2.0"
 toml = "^0.10.0"
 pep440-version-utils = "^0.3.0"
-pydoc-markdown = "^3.5.0"
+pydoc-markdown = "^3.10.0"
```
Comment (m-vdb) [inline pyproject.toml:148]: I've also done [a PR on the base repo](https://github.com/NiklasRosenstein/pydoc-markdown/pull/183) to fix the sorting of the sidebar items, which is currently messed up. Hopefully it gets merged and released in the coming days (cc @tmbo)

**Review by tmbo — APPROVED** [review]
lgtm 💯

**Pull request merged**